### PR TITLE
Remove areas feature

### PR DESCRIPTION
**Pull request by j8seangel**
New area button to remove it, and the areas will be refreshed after the removal.

image

Includes some style refine in the area card after @juancarlosalonso approval.

### CODE REVIEW

File: src/modules/areas.js [hunk 1]
```diff
@@ -99,6 +99,23 @@ export function getArea(id) {
...
+export function deleteArea(areaId) {
+  return (dispatch, state) => {
+    fetch(`${API_BASE_URL}/area/${areaId}`, {
+      headers: {
+        Authorization: `Bearer ${state().user.token}`
+      },
+      method: 'DELETE'
+    })
+      .then(() => {
+        dispatch(getAreas());
```
Comment (sorodrigo) [inline src/modules/areas.js]: Can’t we just remove it from state?

File: src/components/area-card/AreaCard.js [hunk 2]
```diff
@@ -15,11 +20,16 @@ class AreaCard extends React.Component {
...
-                  <Icon className="-small -gray" name="icon-edit"/>
-                </Link>
-              }
+              <div className="area-actions">
+                { user.id === area.userId &&
+                  <Link className="c-button -circle -transparent" to={`/areas/${area.id}`}>
+                    <Icon className="-small -gray" name="icon-edit"/>
+                  </Link>
+                }
+                <button className="c-button -circle -transparent" onClick={this.handleAreaDelete}>
```
Comment (sorodrigo) [inline src/components/area-card/AreaCard.js]: Does team areas appear in this page? If so you’ll need to check for user.id (like in edit), not all users should be able to delete all areas.

File: src/modules/areas.js (hunk 1 above)
Comment (j8seangel) [reply src/modules/areas.js]: Easy peasy, done!

File: src/components/area-card/AreaCard.js (hunk 2 above)
Comment (j8seangel) [reply src/components/area-card/AreaCard.js]: True, I forgot that part 😞 Thanks

**Pull request merged**